### PR TITLE
create most popular section

### LINKED ISSUE #6: create most popular section

### PR DESCRIPTION
**Pull request by JoannaTabbi**
Close #6

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -55,6 +55,66 @@ <h2>Filtres</h2>
           cette ville
         </p>
       </section>
+      <!--section : the most popular-->
+      <section>
+        <div id="most_popular">
```
Comment (Tutt-J) [inline index.html]: remove this div and put id directly on section

File: index.html [hunk 2]
```diff
@@ -55,6 +55,66 @@ <h2>Filtres</h2>
...
+          <!--mp heading-->
+          <div id="most_popular_heading">
```
Comment (Tutt-J) [inline index.html]: remove this div

File: index.html [hunk 3]
```diff
@@ -55,6 +55,66 @@ <h2>Filtres</h2>
...
+            <h2>Les plus populaires</h2>
+            <i class="fas fa-chart-line"></i>
```
Comment (Tutt-J) [inline index.html]: put inside h2

File: index.html [hunk 4]
```diff
@@ -55,6 +55,66 @@ <h2>Filtres</h2>
...
+          </div>
+          <!--mp hotel 1-->
+          <a href="#" class="hotel_card">
```
Comment (Tutt-J) [inline index.html]: remove class

File: index.html [hunk 5]
```diff
@@ -55,6 +55,66 @@ <h2>Filtres</h2>
...
+            <img
+              class="hotel_photo"
+              src="./assets/img/emile-guillemot-Bj_rcSC5XfE-unsplash.jpg"
+              alt="chambre hôtel Le soleil du matin"
+            />
```
Comment (Tutt-J) [inline index.html]: Put inside a figure

File: index.html [hunk 6]
```diff
@@ -55,6 +55,66 @@ <h2>Filtres</h2>
...
+          <div id="most_popular_heading">
+            <h2>Les plus populaires</h2>
+            <i class="fas fa-chart-line"></i>
+          </div>
+          <!--mp hotel 1-->
+          <a href="#" class="hotel_card">
+            <img
+              class="hotel_photo"
+              src="./assets/img/emile-guillemot-Bj_rcSC5XfE-unsplash.jpg"
+              alt="chambre hôtel Le soleil du matin"
```
Comment (Tutt-J) [inline index.html]: Alt must be descritive

File: index.html [hunk 7]
```diff
@@ -55,6 +55,66 @@ <h2>Filtres</h2>
...
+            />
+            <div class="hotel_info">
```
Comment (Tutt-J) [inline index.html]: must be a figaption inside the preview figure, remove class

File: index.html [hunk 8]
```diff
@@ -55,6 +55,66 @@ <h2>Filtres</h2>
...
+              <h3>Hôtel Le soleil du matin</h3>
+              <p>Nuit à partir de 128<strong>€</strong></p>
+              <div class="hotel_score">
+                <i class="fas fa-star"></i><i class="fas fa-star"></i
+                ><i class="fas fa-star"></i><i class="fas fa-star"></i
+                ><i class="fas fa-star"></i>
+              </div>
```
Comment (Tutt-J) [inline index.html]: Use ul with five li for every star and add class "active" on on sould be blue

File: index.html [hunk 9]
```diff
@@ -55,6 +55,66 @@ <h2>Filtres</h2>
...
+            </div>
+          </a>
```
Comment (Tutt-J) [inline index.html]: Put inside a big article

**Review by Tutt-J — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (JoannaTabbi) [reply index.html]: ok

File: index.html (hunk 2 above)
Comment (JoannaTabbi) [reply index.html]: done

File: index.html (hunk 3 above)
Comment (JoannaTabbi) [reply index.html]: done

File: index.html (hunk 4 above)
Comment (JoannaTabbi) [reply index.html]: ok

File: index.html (hunk 5 above)
Comment (JoannaTabbi) [reply index.html]: got it

File: index.html (hunk 6 above)
Comment (JoannaTabbi) [reply index.html]: ok alt description changed, tried to be more "insightful" so tell me if it's ok for you.

File: index.html (hunk 7 above)
Comment (JoannaTabbi) [reply index.html]: done

File: index.html (hunk 8 above)
Comment (JoannaTabbi) [reply index.html]: ok done; do I leave or remove the hotel_score class ?

File: index.html (hunk 9 above)
Comment (JoannaTabbi) [reply index.html]: ok done

File: index.html [hunk 10]
```diff
@@ -55,6 +55,79 @@ <h2>Filtres</h2>
...
+            <figure>
+              <img
+                class="hotel_photo"
+                src="./assets/img/hebergements/3_medium/emile-guillemot-Bj_rcSC5XfE-unsplash.jpg"
+                alt="chambre de l'hôtel style classique avec lit double et mirroir"
+              />
+              <figcaption>
+                <h3>Hôtel Le soleil du matin</h3>
+                <p>Nuit à partir de 128<strong>€</strong></p>
+                <ul class="hotel_score">
```
Comment (Tutt-J) [inline index.html]: Remove class

**Review by Tutt-J — CHANGES_REQUESTED** [review]

**Review by Tutt-J — APPROVED** [review]

**Pull request merged**